codegen.mts

```diff
@@ -51,8 +50,6 @@ const config: CodegenConfig = {
             ProjectAccessScope: '../modules/auth/providers/project-access#ProjectAccessScope',
             OrganizationAccessScope:
               '../modules/auth/providers/organization-access#OrganizationAccessScope',
-            SupportTicketPriority: '../shared/entities#SupportTicketPriority',
-            SupportTicketStatus: '../shared/entities#SupportTicketStatus',
```

Comment:
We are using `enumValues` here, however, I've converted it to use `mappers` since Server Preset already handles mapper logic well enough.

Here's how we add these enums as mappers: https://github.com/kamilkisiela/graphql-hive/pull/4956/files#diff-4b4509fdcb8380696d4a5f77ddebf8d2f6678de05cb598c51edc1f1eb928ff2cR3-R4

Reply:
Actually, this is still needed 😅